### PR TITLE
Update docs from support questions

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

File: content/docs/custom-plugins/deploying/index.md [hunk 1]
```diff
@@ -84,18 +84,25 @@ After the plugin has been uploaded to static hosting, you can follow the documen
 
 ### How do I know which URL to use for the location argument?
 
-When building the plugin for deployment, the most difficult step is usually identifying the correct value for the `--host` argument. If you are building the plugin to be hosted on Roadie, your URL takes the following format:
+When building the plugin for deployment, the most difficult step is usually identifying the correct value for the `--host` argument, or the correct name for the plugin itself. If you are building the plugin to be hosted on Roadie, your URL takes the following format:
```
Comment (Xantier) [inline content/docs/custom-plugins/deploying/index.md:87]:
```suggestion
When building the plugin for deployment, the most difficult step is usually identifying the correct value for the `--host` argument, or the correct name for the plugin path. If you are building the plugin to be hosted on Roadie, your URL takes the following format:
```

File: content/docs/custom-plugins/deploying/index.md [hunk 2]
```diff
@@ -84,18 +84,25 @@ After the plugin has been uploaded to static hosting, you can follow the documen
...
-* The ending (like `my_custom_plugin` above) defines the actual root of the plugin and is usually named after the plugin itself.
+* The ending (like `my_custom_plugin` above) defines the actual root of the plugin and is named after the plugin itself.
 
 Usually you can identify the value of the host by looking at the output when running Roadie CLI in development mode. The default setting for the produced plugin root location will be the same as the `scope` value that Roadie CLI outputs.
 
-If you are hosting the plugin yourself, the URL used in the location argument should match the location where the plugin is reachable from. 
+With the Roadie CLI and the custom plugins automated deploy solution to S3 buckets we are generating the path that **needs** to be used from the plugin name. The plugin name is constructed from the name it is given in the `package.json` file, with few web-safe sanitization added in (dashes replaced with underscores). Unfortunately due to the way the bundling of these assets works, the name needs to be known beforehand and could be tricky to identify correctly.
```
Comment (Xantier) [inline content/docs/custom-plugins/deploying/index.md:96]:
```suggestion
With Roadie CLI and the custom plugins automated deploy solution to Roadie hosted secure solution, we are generating the path that **needs** to be derived from the plugin name. The plugin name is constructed from the name it is given in the `package.json` file, with few web-safe sanitization steps added in (dashes replaced with underscores etc.). Unfortunately due to the way the bundling of these assets works, the name needs to be known beforehand and could be tricky to identify correctly.
```

File: content/docs/custom-plugins/deploying/index.md [hunk 3]
```diff
@@ -84,18 +84,25 @@ After the plugin has been uploaded to static hosting, you can follow the documen
...
 
-If you are hosting the plugin yourself, the URL used in the location argument should match the location where the plugin is reachable from. 
+With Roadie CLI and the custom plugins automated deploy solution to Roadie hosted secure solution, we are generating the path that **needs** to be derived from the plugin name. The plugin name is constructed from the name it is given in the `package.json` file, with few web-safe sanitization steps added in (dashes replaced with underscores etc.). Unfortunately due to the way the bundling of these assets works, the name needs to be known beforehand and could be tricky to identify correctly.
+
+To double check what the name could be, you can take a look at the command output where the upload location is defined (Syncing files ... to `s3://roadie-static-assets/<tenant>/plugin_name`). The **plugin_name** is the correct value that you would need to input as the path during the command running time also if hosting plugins on Roadie. The command run with the following arguments, and corresponding plugin configuration in Roadie, should be able to get the plugin correctly configured:
```
Comment (Xantier) [inline content/docs/custom-plugins/deploying/index.md:98]:
```suggestion
To double check what the name could be, you can take a look at the command output where the upload location is defined (Syncing files ... to `s3://roadie-static-assets/<tenant>/plugin_name`). The **plugin_name** is the correct value that you would need to input as the path during the command run if hosting plugins on Roadie. The command can be run with the following arguments, and corresponding plugin configuration in Roadie, and that should get the plugin correctly configured:
```

File: content/docs/custom-plugins/deploying/index.md [hunk 4]
```diff
@@ -84,18 +84,25 @@ After the plugin has been uploaded to static hosting, you can follow the documen
...
+
+To double check what the name could be, you can take a look at the command output where the upload location is defined (Syncing files ... to `s3://roadie-static-assets/<tenant>/plugin_name`). The **plugin_name** is the correct value that you would need to input as the path during the command run if hosting plugins on Roadie. The command can be run with the following arguments, and corresponding plugin configuration in Roadie, and that should get the plugin correctly configured:
+* `roadie plugin:build --location $PWD/my-plugin-folder --host https://static-assets.roadie.so/<tenant>/plugin_name --withUpload`
+
+Note that the plugin scope in most use cases when hosting in Roadie will be the same.
```
Comment (Xantier) [inline content/docs/custom-plugins/deploying/index.md:101]:
```suggestion
Note that the plugin scope in most use cases when hosting in Roadie will be the same as the path ending needed for the run command.
```

File: content/docs/docs-nav.yaml [hunk 5]
```diff
@@ -61,6 +61,7 @@ nav:
       - Microsoft Graph: '/docs/integrations/ms-graph-org-provider/'
       - Bitbucket: '/docs/integrations/bitbucket/'
       - GitHub Discovery: '/docs/integrations/github-discovery/'
+      - GitHub Discovery Backfill: 'docs/integrations/initial-github-discovery/'
```
Comment (punkle) [inline content/docs/docs-nav.yaml:64]: Is this file already present?

**Review by punkle — APPROVED** [review]

File: content/docs/docs-nav.yaml (hunk 5 above)
Comment (Xantier) [reply content/docs/docs-nav.yaml:64]: Present but not in the nav anywhere

File: content/docs/integrations/github-discovery/index.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ In this example, Roadie will only load a `catalog-info.yaml` file from the `defa
 
 ### Using wildcards
 
-If you would like to catalog a file from all of your repos you can use the \* as a wildcard in any of the segmenets.
+If you would like to catalog a file from all of your repos you can use the \* as a wildcard in any of the segments.
```
[the remark this answers is not in the document]
Comment (Xantier) [reply content/docs/integrations/github-discovery/index.md:64]: I just fixed the typo here, I don't know what `-` is in this context.

**Pull request merged**